hashicorp/dev-portal
# feat: Rewrite docs urls in tutorial content

## ✅ "Ready for Review" checklist




---

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksrewrite-docs-urls-tutorial-content-hashicorp.vercel.app/waypoint/tutorials/get-started-docker/get-started-install) 🔎
- [Asana task](https://app.asana.com/0/1201903760348480/1202144257024022) 🎟️

## 🗒️ What

This PR rewrites docs links in tutorial content. 

## 🛠️ How

Adding onto the remark plugin logic for learn tutorial links, I created a new function to handle docs link rewrites.

## 🧪 Testing

- visit [this tutorial.](https://dev-portal-git-ksrewrite-docs-urls-tutorial-content-hashicorp.vercel.app/waypoint/tutorials/get-started-docker/get-started-install)
- click on a few of the links in the content and make sure you go to the right spot, stay within dev portal
- Click around to [other tutorials](https://dev-portal-git-ksrewrite-docs-urls-tutorial-content-hashicorp.vercel.app/vault/tutorials/getting-started/getting-started-install) and click docs links to make sure they work as expected.
